test/integration/generated/xms-error-responses/Source/XmsErrorResponseExtensionsClient.swift

```diff
@@ -0,0 +1,318 @@
+// --------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+//
+// Code generated by Microsoft (R) AutoRest Code Generator.
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
+// --------------------------------------------------------------------------
+
+import AzureCore
+import Foundation
+// swiftlint:disable superfluous_disable_command
+// swiftlint:disable file_length
+// swiftlint:disable cyclomatic_complexity
+// swiftlint:disable function_body_length
+// swiftlint:disable type_body_length
+
+public final class XmsErrorResponseExtensionsClient: PipelineClient {
+    /// API version of the  to invoke. Defaults to the latest.
+    public enum ApiVersion: String {
+        /// API version ""
+        case v = ""
+
+        /// The most recent API version of the
+        public static var latest: ApiVersion {
+            return .v
```

Comment:
Is this a bug in the code gen?  I guess as long as it works.

Reply:
i think that is because the test swagger does not have api version. I think we can improve code gen by  adding this version block conditionally

Reply:
I'm okay with adding that to this PR or opening an issue to do that as future work.

Reply:
I create https://github.com/Azure/autorest.swift/issues/192